Gems/ROS2/Code/Source/RobotImporter/URDF/URDFPrefabMaker.cpp

```diff
@@ -202,6 +204,7 @@ namespace ROS2
                 {
                     m_status.emplace(name, AZStd::string::format("Failed:  %s", result.GetError().c_str()));
                 }
+                hierarchyMap[AZ::Name(name.c_str())] = childEntity.GetValue();
```

Comment:
Consider `AZ::Name(name.c_str(), name.size())`

Reply:
I did not find this initializer. Are you sure about this?